qemu/tests/cfg/cpu_device_hotpluggable.cfg

```diff
@@ -17,6 +17,11 @@
         required_qemu = [2.12.0, )
     ovmf:
         required_qemu = [5.2.0, )
+        ovmf = yes
```

Comment:
Based on:
https://github.com/avocado-framework/avocado-vt/blob/4b4de70a5cc0a046cee1aaf7bd154da0730c9b0b/virttest/shared/cfg/guest-hw.cfg#L464
https://github.com/avocado-framework/avocado-vt/blob/f3b3a7e7e3507702a30666548f8082751b2e9941/virttest/qemu_devices/qcontainer.py#L1054

I think we can use `ovmf_path` to indicate if it is `ovmf` in test script.